test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -801,13 +798,15 @@ withBlockHeaderDb iordb b = withResource start stop
     stop = closeBlockHeaderDb
 
 withTemporaryDir :: (IO FilePath -> TestTree) -> TestTree
-withTemporaryDir = withResource (fst <$> newTempDir) removeDirectoryRecursive
+withTemporaryDir = withResource
+    (getTemporaryDirectory >>= \d -> createTempDirectory d "test-pact")
+    removeDirectoryRecursive
```

Comment:
This change allows to drop the dependency on the `extra` package and only depend on `temporary`.